plugins/CoreAdminHome/templates/optOut.twig

```diff
@@ -69,5 +86,18 @@
     </form>
     {% endif %}
 {% endif %}
+
+{% if javascripts.external|length > 0 %}
+    {% for script in javascripts.external %}
+        <script type="text/javascript" src="{{ script|raw }}"></script>
```

Comment:
I'm not 100% sure, do we usually output URLs like `{{ script|e('url') }}` see http://twig.sensiolabs.org/doc/filters/escape.html ? Might provide better security but shouldn't be a problem here.